app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', user_confirmation_url(@resource, confirmation_token: @token) %></p>
```

Comment:
- [ ] This is part of the reason why the error displays on the browser because the email generates the wrong Url, you need to remove the `user` from the URL path. :+1: :+1: Here is what the correct URL should look like.
```ruby
<p><%= link_to 'Confirm my account', confirmation_url(@resource, confirmation_token: @token) %></p>
```
- - These should resolve the errors and you can confirm email correctly as seen below:
![Selection_066](https://github.com/Arch-Noize/rorblog/assets/107632780/956dbfd7-eb3d-4ebb-afb2-1fdfa72403ac)

Reply:
I don't know I missed that... thank you so much for pointing me in the right direction.

Reply:
Hi @Arch-Noize, I just tried signing up but after signing up I could see the app without confirming my email

Reply:
Hello @prolajumokeoni. I think I can see why. Inside of my `devise.rb` file there seems to be a configuration that allows for uncomfirmed access for 2 days. I can change that if you wish me to

Reply:
Hi @Arch-Noize,  kindly update it so that the user only has access to your app after they have confirmed their email, please this is a comment not a review 👍🏿